### ISSUE TITLE
Unable to get "hello_wifi" working

### ISSUE THREAD

**Issue by benschinn** [nerves-project/nerves_examples#144]
### Environment

Elixir version 1.9.1 
Nerves environment                                                                                                                                      
  MIX_TARGET:   rpi0                                                                                                                                    
  MIX_ENV:      dev  


### Current behavior
I have cloned the repo, set MIX_TARGET to rpi0, and ran deps.get, firmware, and firmware.burn successfully. 

When I attempt to visit http://192.168.0.1/ via Chrome a connection is never completed. 

I tried to ssh into nerves.local and have run `RingLogger.attach`, and here are pieces that may be relevant:
```
00:00:00.029 [debug] VintageNet(usb0): :configured -> internal configure                                                                                
                                                                                                                                                        
00:00:00.034 [info]  RTC (NervesTime.FileTime) reports that the time is ~N[2020-02-19 00:00:00]                                                         
                                                                                                                                                        
00:00:00.040 [warn]  ntpd crash detected. Delaying next start...                                                                                        
                                                                                                                                                        
00:00:00.137 [info]  VintageNet(wlan0): loading config failed: :enoent                                                                                  
                                                                                                                                                        
00:00:00.137 [info]  VintageNet(wlan0): starting with default config (:no_config)  
```
I have also tried connecting via power to the host (and not usb), but still got the same behavior.

I am very much new to elixir, nerves, and developing embedded programs. please advise, thank you!!

### Expected behavior
Access VintageNet Wizard UI via http://192.168.0.1/ on the browser.

**Comment by benschinn** [comment]
Had conversation with @jjcarstens  via #nerves_dev in slack, and tried running `VintageNetWizard.run_wizard`. Here is the output:
```
{:error,                                                                                                                                                                                                                                                                         
 {:shutdown,                                                                                                                                                                                                                                                                     
  {:failed_to_start_child, :ranch_acceptors_sup,                                                                                                                                                                                                                                 
   {:listen_error, VintageNetWizard.Web.Router.HTTP, :eaddrinuse}}}} 
```
and still failing to connect to http://192.168.0.1/

**Comment by jjcarstens** [comment]
Is there a more full dump of the logs? The small bit listed isn’t quite as helpful. The router seems to have crashed (or failed to start). Something like `RingLogger.grep(~r/error|wizard/i)` would probably be enough. 

After that, can you try `VintageNetWizard.stop_wizard()’ and then ` VintageNetWizard.run_wizard()` again?

**Comment by benschinn** [comment]
starting and stopping and starting again gave me an `:ok`, which made me optimistic, but I am still unable to connect to `http://192.168.0.1/`.
here are the logs I am getting:
```
00:00:12.667 [info]  persistent_ram: uncorrectable error in header                                                                                      
                                                                                                                                                        00:00:12.718 [error] spi-bcm2835 20204000.spi: could not get clk: -517                                                                                  
                                                                                                                                                        00:06:58.236 [error] Failed to start Ranch listener VintageNetWizard.Web.Router.HTTP in :ranch_tcp:listen([cacerts: :..., key: :..., cert: :..., port: 8
0]) for reason :eaddrinuse (address already in use)                                                                                                                                                                                                                                                             
                                                                                                                                                        00:07:07.472 [error] [VintageNetWizard] Failed to run callback: %UndefinedFunctionError{arity: 0, function: :on_wizard_exit, message: nil, module: Munji
gi, reason: nil}                                                                                                                                                                                                                                                                                                
00:07:07.947 [error] brcmfmac: brcmf_vif_set_mgmt_ie: vndr ie set error : -52                                                                                                                                                                                                                                   
00:07:07.957 [error] brcmfmac: brcmf_vif_set_mgmt_ie: vndr ie set error : -52                                                                                                                                                                                                                                   
00:07:07.967 [error] brcmfmac: brcmf_vif_set_mgmt_ie: vndr ie set error : -52                                                                                                                                                                                                                                   
00:07:13.105 [debug] VintageNet(wlan0): :reconfiguring -> call ioctl (returning error)                                                                                                                                                                                                                          
00:07:13.643 [debug] udhcpc(wlan0): udhcpc: read error: Network is down, reopening socket                                                                                                                                                                                                                       
00:07:33.147 [error] brcmfmac: brcmf_vif_set_mgmt_ie: vndr ie set error : -52                                                                                                                                                                                                                                   
00:07:33.148 [error] brcmfmac: brcmf_cfg80211_scan: scan error (-52) 
```

**Comment by jjcarstens** [comment]
Are you connecting to the WiFI access point that it creates? It will show something like `nerves-1234` in your list of WiFi networks.

Starting/stopping that in IEx will also cause the network to disconnect and a lot of machines will automatically connect to another good network if it is around. Prob good to double check you are _actually_ connected to your devices network

**Comment by benschinn** [comment]
> It should show a hello_wifi network that you can connect to.

I understood that I would see `hello_wifi` ssid on the host (my laptop), but now I wondering if I am supposed to be connecting to `hello_wifi` network from my target. If it is the latter case, how do I connect to `hello_wifi` ssid from the target? I cannot connect 192.168.0.1 from the host (and that's where I thought I had to go, but perhaps I misunderstood). Although I do get responses when I `ping "192.168.0.1" ` from the target device.

**Comment by benschinn** [comment]
nevermind the above comment. I figured out the issue. When I had cloned the example repo, I had changes to the config in [target.exs](https://github.com/nerves-project/nerves_examples/blob/master/hello_wifi/config/target.exs#L44). I changed the ssid field to my own wifi ssid at home (hah I thought it to be my own ssid, woops!!), because the ssid already existed it did not work. @jjcarstens thank you for your help and advice!
